lib/parliament/request/url_request.rb

```diff
@@ -1,7 +1,21 @@
 module Parliament
   module Request
+    # API request object, allowing the user to build a request to an API.
```

Comment:
Is this API specific? It's more a way of building URIs based on methods passed to it?